storage_service/locations/tests/test_package.py

```diff
@@ -612,13 +612,20 @@ def test_replicate_aic(self):
             purpose=models.Location.REPLICATOR,
         )
         assert aic.replicas.count() == 0
-        with mock.patch.object(
-            models.Package,
-            "full_pointer_file_path",
+
+        def create_temporary_pointer(path):
+            """Create temporary copy of pointer file at path."""
+            temp_dir = tempfile.gettempdir()
```

Comment:
I notice this class already provides `self.tmp_dir` which is delete on `tearDown`. Could that be used here?

Reply:
Yes, thanks!